src/App.js

```diff
@@ -55,15 +67,19 @@ const useStyles = makeStyles((theme) =>
 
 const App = () => {
   const [image, setImage] = useState();
+  const [metadataDevice, setMetadataDevice] = useState("");
+  const [levelOfAssurance, setLevelOfAssurance] = useState("");
+  const [secureFlag, setSecureFlag] = useState(false);
   const [response, setResponse] = useState();
   const [error, setError] = useState();
 
   const classes = useStyles();
 
-  const onSuccess = ({ image }) => {
-    setImage(image);
+  const onSuccess = ({ img, secure }) => {
+    setImage(img);
+    const fcmResponse = { img, secure };
     service
-      .predict(image)
+      .predict(fcmResponse, levelOfAssurance, metadataDevice)
```

Comment:
I think with this change you can avoid some object mappings in `server.js` and `src/api/api.js` files.
```suggestion
      .predict({ 
        ...fcmResponse,
        level_of_assurance: levelOfAssurance ? levelOfAssurance : undefined,
        metadata: metadataDevice ? { device: metadataDevice } : undefined,
      })
```
What do you think?

Reply:
You both were right :+1: